<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtStopwatchClass</class>
 <widget class="QMainWindow" name="QtStopwatchClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtStopwatch</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>780</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLCDNumber" name="lcd_H_1"/>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd_H_2"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>60</pointsize>
        </font>
       </property>
       <property name="text">
        <string>:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd_M_1"/>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd_M_2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>60</pointsize>
        </font>
       </property>
       <property name="text">
        <string>:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd_S_1"/>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd_S_2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>60</pointsize>
        </font>
       </property>
       <property name="text">
        <string>.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd_ms_1"/>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd_ms_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>781</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_Start">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>START</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Lab">
       <property name="text">
        <string>LAB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Stop">
       <property name="text">
        <string>STOP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Reset">
       <property name="text">
        <string>RESET</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>781</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>829</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtStopwatch.qrc"/>
 </resources>
 <connections/>
</ui>
